ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js

```diff
@@ -688,16 +699,29 @@ module.exports = function CesiumMap(
         pointObject.latitude,
         pointObject.altitude
       )
-      let cartesianPosition = map.scene.globe.ellipsoid.cartographicToCartesian(
+      const image =
+        options.image ||
+        DrawingUtility.getPin({
+          fillColor: options.color,
+          icon: options.icon,
+        })
+      const cartesianPosition = map.scene.globe.ellipsoid.cartographicToCartesian(
```

Comment:
Work done on the image was a necessary bug fix to allow cesium icons to be clickable.